app/controllers/account_reset/cancel_controller.rb

```diff
@@ -11,9 +12,13 @@ def cancel
 
     private
 
-    def handle_success
-      analytics.track_event(Analytics::ACCOUNT_RESET, event: :cancel, token_valid: true)
+    def handle_success(account_reset)
+      user = account_reset.user
+      analytics.track_event(Analytics::ACCOUNT_RESET,
+                            event: :cancel, token_valid: true, user_id: user.uuid)
       sign_out if current_user
+      UserMailer.account_reset_cancel(user.email).deliver_later
+      SmsAccountResetCancelJob.perform_now(phone: user.phone)
```

Comment:
Should this be `perform_later`? Not that it matters while we're using the inline adapter, but thought I'd as since we're using `deliver_later` above.

Reply:
Great catch!  I also need to fix another spot on nil phone.  Rather than a guard clause I'll just check for nil.

Reply:
+1 to only calling the job if the user has a phone, since currently, it's possible for folks to not have a number.